### DISCUSSION TITLE
0x80070490 : Element not found.

### DISCUSSION THREAD

**Discussion by s016aq** [microsoft/winget-cli#4613] [General]
Hi! On a (very) few of our computers I get this message when executing winget with `List/Update/Install`:
```
Failed to open the predefined source; please report to winget maintainers.
An unexpected error occurred while executing the command:
0x80070490 : Element not found.
```
We have 3600+ computers and this appears on 10 of them. (A remediation-script runs on all computers parsing a log for this error. That is how I know which are affected).
Anyone have any idea how to fix it? I have tried everything!
I'm thinking maybe it's not winget that is broken, but maybe these computers have something bad installed causing this?

I have no physical access to these computers and can only connect to them with psexec.

Winget version does not seem to matter. The problem has been since (I think) 1.6.x and know they have 1.8.1791

**Comment by Trenly** [comment]
Have you checked the steps regarding this error in the [troubleshooting guide](https://github.com/microsoft/winget-cli/tree/master/doc/troubleshooting#error-0x80070490)?

> I have no physical access to these computers and can only connect to them with psexec.

WinGet is known to not work with PS Remoting, so that could be related
* https://github.com/microsoft/winget-cli/issues/256

**Comment by s016aq** [comment]
All computers has 23H2, so this KB does not apply to those. Also, not the same error. PS Remoting works fine with all other 3600 computers, so it is not related.
